manifests/params.pp

```diff
@@ -171,9 +171,10 @@
     }
 
     'Debian': {
-      if $::operatingsystem == 'Debian' {
+      if $::operatingsystem == 'Debian' or
```

Comment:
I appreciate a lot of the existing code doesn't, but can we start using the `$facts` hash here instead of using topscope variables?

eg.
```suggestion
      if $facts['operatingsystem'] == 'Debian' or
```

Reply:
(I've not any strong opinion on continuing to use 'legacy facts', but you could use the newer ones like `$facts['os']['name']` instead)

Reply:
Updated to use non-legacy facts from facts hash.